apps/services/search-indexer/src/migrate/index.ts

```diff
@@ -192,11 +214,29 @@ class App {
     logger.info('Elasticsearch migration completed')
     return processedMigrations
   }
+
+  private async migrateKibana() {
+    logger.info('Starting kibana migration')
+    const version = await elastic.getCurrentVersionFromConfig(locales[0])
+    await kibana.importObjects(version)
+    logger.info('Done')
+  }
```

Comment:
I didn't notice any check to prevent this from running multiple times.
This migration script runs in init-container for `search-indexer`.
It's probably not optimal to run this on every init-container, we should look into making this run only when needed

Reply:
I am not very concerned with running this multiple times as all of the json files are combined to a single ndjson file which is imported only once per migration. So this is one API call per pod. The import takes only a second or two.